addons/website_forum/controllers/main.py

```diff
@@ -123,6 +123,7 @@ def questions(self, forum, tag=None, page=1, filters='all', sorting=None, search
         values = self._prepare_user_values(forum=forum, searches=post, header={'ask_hide': not forum.active})
         values.update({
             'main_object': tag or forum,
```

Comment:
Moving edit_in_backend in controller will now show edit in backend for tag.
I don't see any problem with that but if you don't want it tell me, I'll only add it for forum.

Reply:
Only for forum, done